### PR TITLE
[i18n] Update arabic

### PR DESCRIPTION
**Pull request by faz1993**
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/i18n/locales/ar.js [hunk 1]
```diff
@@ -129,8 +129,8 @@ export default {
 	creating_invite: 'إنشاء دعوة',
 	Channel_Name: 'اسم القناة',
 	Channels: 'قنوات',
-	Chats: 'محادثات',
-	Call_already_ended: 'انتهت المكالمة بالفعل!',
+	Chats: 'الرسائل',
+	Call_already_ended: 'انتهت المكالمة!',
```
Comment (diegolmello) [inline app/i18n/locales/ar.js:133]: I'm reviewing with Google Translate, so it's not accurate, but it looks like the current translation is better.
The intention here is to tell the user "The call has already ended", which is precisely what the current string says.

File: app/i18n/locales/ar.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ export default {
 	Announcement: 'إعلان',
 	Apply_Your_Certificate: 'طبق شهادتك',
 	Applying_a_theme_will_change_how_the_app_looks: 'سيؤدي تطبيق السمة إلى تغيير شكل التطبيق',
-	ARCHIVE: 'أرشفة',
+	ARCHIVE: 'أرشيف',
```
Comment (diegolmello) [inline app/i18n/locales/ar.js:104]: This is a call to action to `Archive` a channel.
It's not a list of things.

**Review by diegolmello — CHANGES_REQUESTED** [review]
Can you take a look? Thanks!

File: app/i18n/locales/ar.js (hunk 1 above)
Comment (faz1993) [reply app/i18n/locales/ar.js:133]: sometimes google give wrong translation in arabic it well the call has already ended in first translation was the call is really ended but in the string is the call already ended so i corrected it to the same meaning

File: app/i18n/locales/ar.js (hunk 2 above)
Comment (faz1993) [reply app/i18n/locales/ar.js:104]: ahhh i tought its list

**Review by diegolmello — APPROVED** [review]

**Pull request merged**